packages/create-vite/template-react-ts/package.json

```diff
@@ -16,8 +16,8 @@
   "devDependencies": {
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
-    "@typescript-eslint/eslint-plugin": "^7.15.0",
-    "@typescript-eslint/parser": "^7.15.0",
+    "@typescript-eslint/eslint-plugin": "8.0.0-alpha.39",
+    "@typescript-eslint/parser": "8.0.0-alpha.39",
```

Comment:
I think I'd prefer using `^` even if they're alpha versions so users can get updates as soon as it's stable too. Do you foresee more breaking changes? If so, perhaps we should keep the templates v7, otherwise it would be nice to use `^` here.